.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity and is not currently on maintainers' radars. It will be closed
+  in a week if no further activity occurs :)
+
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```

Comment:
When an issue is closed I think we should say something like "Please leave a comment if this issue is still relevant."